sdk/tests/wallet/wallet_storage.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{fs, io, path::Path};
+
+use iota_sdk::wallet::{Result, Wallet};
+
+use crate::wallet::common::{setup, tear_down};
+
+#[cfg(all(feature = "stronghold", feature = "rocksdb"))]
+#[tokio::test]
+async fn check_existing_db() -> Result<()> {
+    let storage_path = "check_existing_db_test";
+    setup(storage_path)?;
+    // Copy db so the original doesn't get modified
+    copy_folder("./tests/wallet/fixtures/check_existing_db_test", storage_path).unwrap();
+
+    let manager = Wallet::builder().with_storage_path(storage_path).finish().await?;
+
+    // Test if setting stronghold password still works
+    manager.set_stronghold_password("STRONGHOLD_PASSWORD").await?;
```

Comment:
lets move this password to `constants.rs`

Reply:
But it's specific do this db, why would you want it somewhere else?